Source/WTF/wtf/text/StringImpl.h

```diff
@@ -1324,6 +1327,46 @@ inline Ref<StringImpl> StringImpl::removeCharacters(const Predicate& findMatch)
     return removeCharactersImpl(characters16(), findMatch);
 }
 
+inline Ref<StringImpl> StringImpl::createByReplacingInCharacters(const LChar* characters, unsigned length, UChar target, UChar replacement, unsigned indexOfFirstTargetCharacter)
+{
+    ASSERT(indexOfFirstTargetCharacter < length);
+    if (isLatin1(replacement)) {
+        LChar* data;
+        LChar oldChar = static_cast<LChar>(target);
+        LChar newChar = static_cast<LChar>(replacement);
```

Comment:
Why are these static_cast needed? It’s great to get a warning when narrowing, but I don’t think there is one and so I don’t think these casts are needed. The local variables are, but not the casts.